src/java/org/apache/cassandra/transport/CQLMessageHandler.java

```diff
@@ -157,41 +160,97 @@ protected boolean processOneContainedMessage(ShareableBytes bytes, Limit endpoin
 
         // max CQL message size defaults to 256mb, so should be safe to downcast
         int messageSize = Ints.checkedCast(header.bodySizeInBytes);
+        
         if (throwOnOverload)
```

Comment:
Might want to comment here on what we talked about offline, that there's an expected "work anyway then back pressure" vs. "don't process and throwOnOverload" delta between the two modes of operation and how they deal with too much inbound.